src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
+    }
+
+    public Ball(String color, int number) {
+        this.ballColor = color;
+        this.ballNumber = number;
+    }
+
+    public String getBallColor() {
+        return this.ballColor;
```

Comment:
Its unnecessary to use 'this' in getter, also you can use @Getter @Setter annotation from Lombok on class-level annotations. More information you can read here: https://projectlombok.org/features/GetterSetter

Reply:
Your code is concise and well-structured. Using 'this' to access class fields is a good practice that enhances code clarity. Great job!